src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor

```diff
@@ -102,7 +102,10 @@
                                 }
                             </TemplateColumn>
                             <TemplateColumn Title="@ControlsStringsLoc[nameof(ControlsStrings.DetailsColumnHeader)]" Class="no-ellipsis">
-                                <FluentButton Appearance="Appearance.Lightweight" OnClick="() => OnShowProperties(context)">@ControlsStringsLoc[nameof(ControlsStrings.ViewAction)]</FluentButton>
+                                @{
+                                    var id = $"details-button-{context.GetHashCode()}";
```

Comment:
GetHashCode isn't unique. Probably the best thing is to add an InternalId property of guid to the type and have it assigned when the data is received.